coffee/lib/abstract-chosen.coffee

```diff
@@ -282,12 +282,20 @@ class AbstractChosen
   # class methods and variables ============================================================
 
   @browser_is_supported: ->
-    if window.navigator.appName == "Microsoft Internet Explorer"
-      return document.documentMode >= 8
     if /iP(od|hone)/i.test(window.navigator.userAgent)
       return false
     if /Android/i.test(window.navigator.userAgent)
       return false if /Mobile/i.test(window.navigator.userAgent)
+    if /IEMobile/i.test(navigator.userAgent)
```

Comment:
It would still work, would it not? window being the global object and all. But I'll add it for consistency.